pkg/controllers/clusterresource/keymanagementprovider_controller.go

```diff
@@ -85,7 +83,7 @@ func (r *KeyManagementProviderReconciler) Reconcile(ctx context.Context, req ctr
 		logger.Warn("Certificate Store already exists. Key management provider and certificate store should not be configured together. Please migrate to key management provider and delete certificate store.")
```

Comment:
Does it make sense to do this check to make sure certstore + kmp are not configured together? Certstore is namespaced and KMP is both cluster and namespaced. So then cluster KMP will never overlap with namespaced cert store?

Reply:
I feel NamespacedKMP and KMP should be regarded as a cohesive unit. And since we're just warning which is a reminder for users to migrate but not blocking certStore operations.